docs/_source/guides/llms/practical_guides/fine_tune_others.md

```diff
@@ -61,12 +61,12 @@ task_mapping = TrainingTaskMapping.for_text_classification(
 
 :::{tab-item} LabelQuestion
 ```python
-from argilla import FeedbackDataset, TrainingTaskMapping
+import argilla.feedback as rg
```

Comment:
Do we want to import this with the same alias as `import argilla as rg`?

I think can be confusing for user when they want mix feedback trainings opérations with other ones.

I would say just import required classes normally

```
from argilla.feedback import FeedbackDataset, TrainingTaskMapping
```

Reply:
@frascuchon, I changed this.